Initial Code Review (#3)
dataiku/dss-plugin-generalized-linear-models: README.md

# Generalized Linear Model Plugin
The GLM plugin allows users to implement generalized linear model functionality within DSS visual machine learning.
Binary Classification and Regression are available but multiclass is not. The models use the same logic as the 

> [statsmodels library](https://www.statsmodels.org/stable/glm.html). Please note this plugin is still in development 

Reviewer: ```suggestion
[statsmodels library](https://www.statsmodels.org/stable/glm.html). Please note this plugin is not supported 
```

Reviewer 2: @david-behar

Reviewer 3: [46109b1](https://github.com/dataiku/dss-plugin-generalized-linear-models/commit/46109b137399b717cb19b3179d8044a81f85cc07)

Note this plugin is currently in development and its accuracy cannot be guaranteed 
# Generalized Linear Model Plugin
The GLM plugin allows users to implement generalized linear model functionality within DSS visual machine learning 

GLMs in the doctor are  not currently feasible as the doctor is not compatible with newer version of sklearn. 
This plugin writes the statsmodel glms in the sklearn framework and is now compatible.

# Implemented

1. Binary Classification Model
2. Regression Model

# To Do

1. Implement Multiclass GLMs
2. Implement grid search strategy
 - Known issue: 3 hyper parameters are used for grid search (Power, Var Power, Alpha), 
   at any time the max a model will need defined is two hyper parameters, but it still tries to optimise based on third which causes an error as there is no change in performance.
   Solution 1: Custom grid search

>    Solution 2: Make it not essential to define hyper parameters, so they are ignored in grid search

Author: reformatted read me (fb0744d02101667ee169f5b3b27e7ff9de1c137a) and added change log (c138205b70109ec2d1de9b93abb7e6e305843998). For the alpha release we will not include documentation on the website.

>    Solution 2: Make it not essential to define hyper parameters, so they are ignored in grid search

Reviewer: Can you add a link to the changelog in the README and content inside the changelog? See https://github.com/dataiku/dss-plugin-timeseries-forecast/blob/main/README.md for an example

>    Solution 2: Make it not essential to define hyper parameters, so they are ignored in grid search

Author: 67f93aacd60a639110b88f6a57a77e12ffc4fbf0

>    Solution 2: Make it not essential to define hyper parameters, so they are ignored in grid search

Reviewer: Need to put something on the changelog :)

>    Solution 2: Make it not essential to define hyper parameters, so they are ignored in grid search

Reviewer 3: [3f97c4b](https://github.com/dataiku/dss-plugin-generalized-linear-models/commit/3f97c4bec7dd44bfc98294d3b94c8b19e05e04ae)
